client/src/components/PortfolioFilters.tsx

```diff
@@ -142,22 +152,18 @@ export const PortfolioFilters = React.memo(
             isActive={ownernamesActive}
             isOpen={ownernamesIsOpen}
             setIsOpen={setOwnernamesIsOpen}
-            initialFocus={isMobile ? undefined : "#filter-ownernames-multiselect_input"}
+            initialFocus={isMobile ? undefined : "#filter-ownernames-multiselect input"}
             selectionsCount={filterContext.filterSelections.ownernames.length}
             className="ownernames-accordion"
+            i18n={i18n}
```

Comment:
could the i18n just be imported in the `FilterAccordion` file rather than being passed as a prop? same with `MultiSelect` component below